locales/en/app.ftl

```diff
@@ -514,6 +514,23 @@ want-to-add = Want to add another email?
 # the form to add an additional email to Firefox Monitor.
 verify-the-link = Verify the link sent to { $userEmail } to add it to {-product-name}.
 
+## These are part of a confirmation page that appears after a user has verified
+## an additional email to Firefox Monitor.
+
+email-verified = Email Successfully Verified!
+email-added-to-subscription = We’ll alert you if { $email } appears in a data breach.
+
+# This is the first part of the message that appears after the user has verified
+# their email address. It reads "To see and manage all emails you've signed up
+# for breach monitoring, sign in.
+email-verified-view-dashboard = To see and manage all emails you’ve signed up for breach monitoring, { $nestedSignInLink }.
+# This is the second part of the message that appears after the user has verified
+# their email address. It reads "To see and manage all emails you've signed up
+# for breach monitoring, sign in.
+sign-in-nested = sign in.
```

Comment:
I'm struggling to piece this together, but I want to say that I object to the trailing period in this string.
If I'm reading it correctly, it'd do something like double period at the end of the sentence (one linked, one not):

```
To see and manage all emails you've signed up for breach monitoring,
<a class="text-link bold" href="/oauth/init">sign in.</a>.
```

Reply:
good catch! updated.